pkg/plugin/manager.go

```diff
@@ -0,0 +1,196 @@
+package plugin
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"log"
```

Comment:
normally we shouldn't have to import the 2 packages above, as we use afero and logrus.